File: app/src/main/java/emperatriz/riverflood/model/Evento.java
package emperatriz.riverflood.model;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.Date;

/**
 * Created by ramon on 24/04/2017.
 */



public class Evento implements Serializable {

    public static final int FUTBOL=0, BALONCESTO=1, F1=2, MOTOS=3, TENIS=4, DESCONOCIDO=5, CICLISMO=6, BOXEO=7, ATLETISMO=8;

    public String nombre, competicion, url;
    public int tipo, fondo;
    public Date hora;
    public boolean directo;
    public ArrayList<Link> links;
}

File: app/src/main/java/emperatriz/riverflood/model/Evento.kt
package emperatriz.riverflood.model

import java.io.Serializable
import java.util.ArrayList
import java.util.Date

/**
 * Created by ramon on 24/04/2017.
 */


class Evento : Serializable {

    var nombre: String? = null
    var competicion: String? = null
    var url: String? = null
    var tipo: Int = 0
    var fondo: Int = 0
    var hora: Date? = null
    var directo: Boolean = false
    var links: ArrayList<Link>? = null

    companion object {

        val FUTBOL = 0
        val BALONCESTO = 1
        val F1 = 2
        val MOTOS = 3
        val TENIS = 4
        val DESCONOCIDO = 5
        val CICLISMO = 6
        val BOXEO = 7
        val ATLETISMO = 8
    }
}
